### PR TITLE
roles: add role to deploy metric aggregation rules

### PR DESCRIPTION
**Pull request by fromanirh**
Using PrometheusRules to configure the aggregation rules,
we can create the metrics CNV wants to consume out of the
metrics that kubevirt provides out of the box.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: roles/KubevirtMetricsAggregator/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+# defaults file for KubevirtCommonTemplatesBundle
```
Comment (fromanirh) [inline roles/KubevirtMetricsAggregator/defaults/main.yml]: rename

File: roles/KubevirtMetricsAggregator/templates/aggregation-rule-vmi-count.yaml.j2 [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    prometheus: k8s
+    role: cnv-rules
+  name: prometheus-k8s-rules-cnv
+  namespace: "{{ meta.namespace }}"
+spec:
+  groups:
+  - name: cnv.rules
+    rules:
+    - expr: |
+        sum(kubevirt_vmi_phase_count{phase="running}) by (node)
```
Comment (fromanirh) [inline roles/KubevirtMetricsAggregator/templates/aggregation-rule-vmi-count.yaml.j2]: typo

**Comment by ksimon1** [comment]
LGTM
just one thing, do we need these files? 
roles/KubevirtMetricsAggregation/defaults/main.yml
roles/KubevirtMetricsAggregation/meta/main.yml
roles/KubevirtMetricsAggregation/handlers/main.yml

**Comment by fromanirh** [comment]
> LGTM
> just one thing, do we need these files?
> roles/KubevirtMetricsAggregation/defaults/main.yml
> roles/KubevirtMetricsAggregation/meta/main.yml
> roles/KubevirtMetricsAggregation/handlers/main.yml

We don't, but not sure the ansible role layout needs them. If not we can cleanup this and other roles, will do in a separate patch.

**Pull request merged**